includes/classes/PluginSupport/ScriptedInstallHelpers.php

```diff
@@ -103,12 +104,12 @@ public function updateConfigurationKey(string $key_name, array $properties): int
         return $rows;
     }
 
-    public function deleteConfigurationKeys(array $key_names): int
+    protected function deleteConfigurationKeys(array $key_names): int
     {
         if (empty($key_names)) {
             return 0;
         }
-        $keys_list = implode("','", array_map(fn ($val) => zen_db_input($val), $key_names));
+        $keys_list = implode("','", array_map(static fn ($val) => \zen_db_input($val), $key_names));
```

Comment:
In that case it's because it's inside a sub function, and the $this variable isn't in scope

Reply:
Additionally, there's no error-trapping with that method call anyway, so moot as far as that topic goes.

Reply:
OK, but how about some consistency given that line 98 uses the `$this->dbConn` form.

Reply:
Right. Consistency is throughtout, where it can be used: The rest of the file is using the $this->dbConn format. 
This isolated one, since it's inside a separate function, has no access to the parent/external `$this` object, which is why we're forced to use the function call instead.

Reply:
The closest to being more "consistent" would be the following, but it still breaks pattern, and creates a new object to do it:
```diff
+        $db = $this->dbConn;
-        $keys_list = implode("','", array_map(static fn ($val) => zen_db_input($val), $key_names));
+        $keys_list = implode("','", array_map(static fn ($val) => $db->prepare_input($val), $key_names));
```